drgn/helpers/linux/sched.py

```diff
@@ -163,3 +164,44 @@ def loadavg(prog: Program) -> Tuple[float, float, float]:
     avenrun = prog["avenrun"]
     vals = [avenrun[i].value_() / (1 << 11) for i in range(3)]
     return (vals[0], vals[1], vals[2])
+
+
+def cpu_rq(prog: Program, cpu: int) -> Object:
```

Comment:
This should use `drgn.helpers.common.prog.takes_program_or_default` and `drgn.IntegerLike` to make the `prog` argument optional:

```suggestion
@takes_program_or_default
def cpu_rq(prog: Program, cpu: IntegerLike) -> Object:
```